codice/ddf
# DDF-04267 Upgraded Geotools and Associated Dependencies

#### What does this PR do?
Implements upgraded geotools dependencies from thirdparty. 

#### Who is reviewing it? 

@bakejeyner 
@pvargas 
@rfding 
@aperullo 

#### Ask 2 committers to review/merge the PR and tag them here.
@oconnormi
@stustison

#### How should this be tested?
Run a full build with these changes. Verify that the build passes.

For GH Issues:
Fixes: #__04267__

#### Checklist:

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
